### ISSUE TITLE
Menus are anounced as submenus in apps like notepad

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4982]
**Reported by sumandogra on 2015-03-18 10:42**
In notepad, menus are announced as submenus by NVDA.  This is a confusion for a usual layman user who uses RIGHT ARROW to open the submenus.
.

Steps to reproduce:
1.  Open notepad.
2.  Press ALT to go to the menu bar.
3.  Use RIGHT ARROW to navigate in the menu bar.  File, edit, view meus are announced as submenus.

**Comment by bhavyashah** [comment]
@jcsteh By https://github.com/nvaccess/nvda/issues/4982#issuecomment-155335141, are you trying to imply that this ticket should be closed as a wontfix? It may be worth noting that Narrator reads what NVDA calls 'File submenu" as 'File collapsed menu item". CC @sumandogra

**Comment by jcsteh** [comment]
These are menu items which open menus, not menus themselves. Closing as won't fix.
